src/server/webhooks/events/pr/reviewer_updated.ts

```diff
@@ -44,11 +64,12 @@ export interface PullRequest {
 	readonly draft: boolean
 	readonly fromRef: Ref
 	readonly id: number
+	readonly links: ActorLinks
```

Comment:
Probably, but those types are auto-generated from Quicktype and Atlassian's docs. I'm keeping them minimally changed to avoid maintenance burden later on.